packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -295,7 +308,8 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
         };
 
         this.connectionManager = createConnectionManager(props);
-
+        this.mc = loggerToMonitoringContext(logger);
+        this.preventConcurrentOpSend = this.mc.config.getBoolean("Fluid.Container.ConcurrentOpSend") ?? false;
```

Comment:
nit: I think the output JS is more concise if you do `== true` rather than `?? false`

Reply:
You can check in typescript playground to quickly compare the JS output.